NGitLab/IIssueClient.cs

```diff
@@ -134,6 +134,14 @@ public interface IIssueClient
         /// <returns>The list of MR that are related this issue.</returns>
         IEnumerable<MergeRequest> RelatedTo(int projectId, int issueIid);
 
+        /// <summary>
+        /// Get all Issues that are linked to a particular issue of particular project.
+        /// </summary>
+        /// <param name="projectId">The project id.</param>
+        /// <param name="issueId">The id of the issue in the project's scope.</param>
+        /// <returns>The list of Issues linked to this issue.</returns>
+        IEnumerable<Issue> LinkedTo(int projectId,int issueId);
```

Comment:
You should use the newer type `GitLabCollectionResponse` which allow sync and async usage.

```suggestion
        GitLabCollectionResponse<Issue> LinkedTo(int projectId,int issueId);
```

Reply:
BTW, the new ```LinkedTo``` uses the same pattern as ```RelatedTo```:
![image](https://github.com/ubisoft/NGitLab/assets/36262995/abd2fc5a-2e59-4e54-9c87-9b8fbea443a7)

Reply:
The GitLabCollectionResponse is recent, so you can see lots of methods that still use the old pattern.